src/atlas-orca/meshgenerator/OrcaMeshGenerator.cc

```diff
@@ -180,7 +180,7 @@ void OrcaMeshGenerator::generate( const Grid& grid, const grid::Distribution& di
 
     //---------------------------------------------------
 
-    if ( serial_distribution ) {
+    if ( serial_distribution && (halosize_ == 0) ) {
```

Comment:
if a serial distribution the halosize should necessarily be zero. I was going to put a guard for this in the meshgenerator setup if there isn't one already.  I also removed this case from the LocalOrcaGrid testing.

Reply:
Thanks for the explanation - I'll change this back.